packages/mdc-ripple/util.js

```diff
@@ -97,31 +97,46 @@ function supportsCssVariables(windowObj, forceRefresh = false) {
  * Determine whether the current browser supports passive event listeners, and if so, use them.
  * @param {!Window=} globalObj
  * @param {boolean=} forceRefresh
- * @return {boolean|{passive: boolean}}
+ * @return {boolean|!EventListenerOptions}
  */
 function applyPassive(globalObj = window, forceRefresh = false) {
   if (supportsPassive_ === undefined || forceRefresh) {
     let isSupported = false;
     try {
       globalObj.document.addEventListener('test', null, {get passive() {
         isSupported = true;
+        return isSupported;
```

Comment:
Adding this here is going to skip caching the result, which is done below.

Reply:
The return here is in the context of the getter. I added this to solve for cases where the type system requires all getters to return a value.

Reply:
Oh whoops, sorry, I misread. I see what you mean.